.github/workflows/pr-build.yml

```diff
@@ -40,4 +40,16 @@ jobs:
       - name: Build with Gradle
         uses: hypertrace/github-actions/gradle@main
         with: 
-          args: dockerBuildImages
\ No newline at end of file
+          args: dockerBuildImages
+
+      - name: Determine docker tag
+        id: tag
+        run: echo ::set-output name=tag::$(./gradlew -q printDockerImageDefaultTag | head -1)
+
+      - name: Scan docker image
+        uses: azure/container-scan@v0.1
+        with:
+          image-name: hypertrace/java:${{ steps.tag.outputs.tag }}
```

Comment:
tags are hard-code in this repo.
- hypertrace/java:11
- hypertrace/java:14

Need to scan above two images.

Reply:
okay ill edit that